diff -Nru gbatnav-0.74.0/gbnserver/Makefile.am gbatnav-0.74.0.new/gbnserver/Makefile.am
--- gbatnav-0.74.0/gbnserver/Makefile.am	Sat Apr  3 02:37:42 1999
+++ gbatnav-0.74.0.new/gbnserver/Makefile.am	Sun Oct 29 12:10:23 2000
@@ -48,9 +48,9 @@
 Games_DATA = gbnserver.desktop
 
 install-data-local:
-	-$(mkinstalldirs) $(scoredir) $(helpdir)
-	-touch $(scoredir)/gbatnav.scores
-	-chown games.games $(scoredir)/gbatnav.scores
+	-$(mkinstalldirs) $(DESTDIR)$(scoredir) $(DESTDIR)$(helpdir)
+	-touch $(DESTDIR)$(scoredir)/gbatnav.scores
+	-chown games.games $(DESTDIR)$(scoredir)/gbatnav.scores
 	-chmod 664 $(scoredir)/gbatnav.scores
-	-$(INSTALL_DATA) $(srcdir)/index.html $(helpdir)
-	-$(INSTALL_DATA) $(srcdir)/gbatnav-topic.dat $(helpdir)/topic.dat
+	-$(INSTALL_DATA) $(srcdir)/index.html $(DESTDIR)$(helpdir)
+	-$(INSTALL_DATA) $(srcdir)/gbatnav-topic.dat $(DESTDIR)$(helpdir)/topic.dat
